src/shared/components/annotation/Civic.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {Modal} from 'react-bootstrap';
+import {observer} from "mobx-react";
+import {Circle} from "better-react-spinkit";
+import DefaultTooltip from 'shared/components/DefaultTooltip';
+import annotationStyles from "./styles/annotation.module.scss";
+import {ICivicVariant, ICivicEntry} from "shared/model/Civic.ts";
+import {observable} from "mobx";
+import CivicCard from "./CivicCard";
+
+export interface ICivicProps { 
+    civicEntry: ICivicEntry | null | undefined;
+    isCivicDisabled: boolean;
```

Comment:
Let's remove this property (and the related code segment) from the component, and assume that if it is rendered, it is enabled.